src/v/utils/rjson_allocator.h

```diff
@@ -0,0 +1,26 @@
+#include <rapidjson/allocators.h>
+#include <rapidjson/document.h>
+#include <rapidjson/encodings.h>
+
+namespace rjson_allocator {
```

Comment:
I think this file should be named `src/v/json/allocator.h` and the namespace `json`.